### DISCUSSION TITLE
Unable to run Sonarqube - Automatically stopping and going to maintenance

### DISCUSSION THREAD

**Discussion by LokeshNagireddy** [learndevopsonline/learndevopsonline#1900] [Q&A]
Hi Team,

Im Lokesh from Batch 72.
While im practicing code quality as part of Jenkins multi branch pipeline, i have installed sonarqube in a separate server through labauto. My server is a spot instance.

When i try to open it, its automatically stopping and going into maintenance page. Thereafter, the server is getting freezed and i cant access it in even putty.

Shared the screen shots in the attachment. Can someone please help me to solve the issue.

i look forward for your response.
[sonarqube errors.docx](https://github.com/learndevopsonline/learndevopsonline/files/15418001/sonarqube.errors.docx)

**Comment by chaithanya1812** [comment]
### Issue is resolved in lab session
